### PR TITLE
Flask Service

### PR DESCRIPTION
**Pull request by johngalimi**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    return jsonify(query_result)
+
+
+@app.route("/leagues/<league>/teams/<team>/games")
+def games_by_league_and_team(league,team):
+    date = request.args.get("date")
+
+    if date is None:
+        return "Please pass a date query parameter in the format YYYY-MM-DD."
```
Comment (johngalimi) [inline src/app.py:59]: skip a line between this if block and the query, same thing on the other endpoint.

File: src/app.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+def games_by_league(league):
+    date = request.args.get("date")
+
+    if date is None:
+        return "Please pass a date query parameter in the format YYYY-MM-DD."
+    query =f"""
+    SELECT
+        league.league_abbr,
+        game.game_season,
+        game.game_date,
```
Comment (johngalimi) [inline src/app.py]: Do we need to pull the column `game.game_date` if we're filtering to a single date?

File: src/app.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        ON game.team_id = team.id
+    INNER JOIN tblTeam opponent
+        ON game.opponent_id = opponent.id
+    INNER JOIN tblLeague league
+        ON team.league_id = league.id
+    WHERE league.league_abbr = '{league}'
+    AND team.team_abbr = '{team}'
+    AND game.game_date = '{date}'  
+    """
+    query_result = execute_query(query)
```
Comment (johngalimi) [inline src/app.py:80]: skip line above this for readability (same in the other endpoint)

File: src/app.py (hunk 2 above)
Comment (johngalimi) [reply src/app.py]: same for league_abbr

Comment (jdg24) [reply src/app.py]: addressed Thanks

**Pull request merged**